images/runtime/dotnetcore/DockerfileWithCurlUpdate.template

```diff
@@ -16,7 +16,12 @@ FROM %DOTNETCORE_BASE_IMAGE%
 # We manually update it here so we can still depend on the original images.
 # This command should be removed once support for deprecated .NET core images is halted.
 RUN sed -i '/jessie-updates/d' /etc/apt/sources.list  # Now archived
-RUN apt-get update && apt-get install curl --yes
```

Comment:
how about we just update `curl` in all images, since you're adding an apt install command? That would free us from maintaining those two templates.

Reply:
Good point 👍 ..will do that

Reply:
Oh seems like we are also doing this in the image having the curl update
```
# Older .NET core versions, which have reached end of life and therefore are no longer updated, use
# a version of `curl` that has known issues.
# We manually update it here so we can still depend on the original images.
# This command should be removed once support for deprecated .NET core images is halted.
RUN sed -i '/jessie-updates/d' /etc/apt/sources.list  # Now archived
```
So probably we cannot move it(?)